packages/blade/src/components/Carousel/CarouselItem.web.tsx

```diff
@@ -34,7 +34,7 @@ const StyledCarouselItem = styled(BaseBox)<StyledCarouselItemProps>(
       ...(isResponsive && {
         width: '100%',
         scrollSnapAlign: isMobile || !shouldAddStartEndSpacing ? 'start' : 'center',
-        marginLeft: shouldHaveStartSpacing ? '100%' : 0,
+        marginLeft: shouldHaveStartSpacing ? '40%' : 0,
```

Comment:
what was the reason of this miss? was it scroll gesture?

Reply:
This only happens if you use trackpad to purposely scroll past the left side too much. 

Otherwise not an experience breaker.